UnityProject/Assets/Scripts/Items/Medical/HealsTheLiving.cs

```diff
@@ -77,6 +78,10 @@ public virtual void ServerPerformInteraction(HandApply interaction)
 
 	private void ServerApplyHeal(LivingHealthMasterBase livingHealth, HandApply interaction)
 	{
+		Chat.AddActionMsgToChat(interaction,
+			$"You apply the {this.gameObject.ExpensiveName()} to {livingHealth.gameObject.ExpensiveName()} Healing them.".Color(Color.green),
```

Comment:
heal them*

Reply:
healing them because you're performing the action